app/util/Tracing.js

```diff
@@ -0,0 +1,331 @@
+/**
+ * Util class for tracing related functions.
+ *
+ * @class CpsiMapview.util.Tracing
+ */
+Ext.define('CpsiMapview.util.Tracing', {
+
+    statics: {
+
+        /**
+         * Compute the modulo for negative values.
+         *
+         * Example: staticMe.computeModulo(-1, 4) returns 3
+         *
+         * Taken from the official OpenLayers example:
+         * https://openlayers.org/en/latest/examples/tracing.html
+         *
+         * @param {Number} a The first number
+         * @param {Number} b The second number
+         *
+         * @returns {Number} The modulo
+         */
+        computeModulo: function (a, b) {
+            return ((a % b) + b) % b;
+        },
+
+        /**
+         * Checks if a LineString is not empty.
+         *
+         * @param {ol.Feature} feature The feature to check
+         * @returns {Boolean} If LineString is not empty
+         */
+        lineStringNotEmpty: function (feature) {
```

Comment:
Minor: It might be easier to work with functions that return booleans, if the function name does not already include a negation, since these functions will probably be used with other logical operator in e.g. if-statements.

So a `lineStringEmpty` or a `lineStringPopulated` will be easier to work with, as we will not have a doubled negation.

Compare

```js
if (!lineStringEmpty) {
```

to

```js
if (!lineStringNotEmpty) {
```

Reply:
good point! done with 2c936b1